http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/HttpServletWithSessionListenerIT.java

```diff
@@ -51,6 +57,30 @@ public void sessionSecured() {
         thenCheckActiveSessionsEqualTo(1);
     }
 
+    @Test
+    @DisplayName("undertow servlet test http2")
```

Comment:
Undertow is a name, so it should be capitalized. I'm also not sure http2 is according to the convention (HTTP/2)